src/xmipp/libraries/data/ctf.h

```diff
@@ -27,7 +27,8 @@
 #define _CORE_CTF_HH
 
 #include <complex>
-#include "core/metadata.h"
+#include "core/metadata_vec.h"
```

Comment:
Both includes are necessary?
Can they be included in the cpp file instead of the header?

Reply:
Only db include in reqiured, vec removed.